tensorflow/core/util/device_name_utils.cc

```diff
@@ -162,6 +162,16 @@ bool DeviceNameUtils::ParseFullName(StringPiece fullname, ParsedName* p) {
       }
       progress = true;
     }
+    if (str_util::ConsumePrefix(&fullname, "/sycl:") ||
```

Comment:
You can remove this: device assignments can just use "/device:SYCL:0" instead and you don't need a special-case here